backend/curriculum_tracking/admin.py

```diff
@@ -141,8 +140,9 @@ class UserAdmin(BaseUserAdmin):
         # GroupInline,
         # RecruitProjectInline,
     )
-    list_display = ("email", "is_superuser")
+    list_display = ("email", "is_superuser", "active")
     list_filter = (
+        "groups",
```

Comment:
There are currently 294 groups in the Tilde prod database. Will this put all 294 groups here? 

![image](https://github.com/Umuzi-org/Tilde/assets/699166/f5a93476-6962-444d-ac8e-861c114afba5)

That would be a lot of data - it'll be slow, most of the time it wont be needed, it'll also make the other filters less accessable.  

This filter should be removed.

If there is a better way to put a filter in here that would be good. Eg if there was a textbox or something then that would be usable. But I'm not sure how hard that is to pull off.